app/components/works/dates_component.html.erb

```diff
@@ -60,21 +60,22 @@
         <div class="year">
           <label for="<%= prefix %>_created_range_start_year" class="visually-hidden">Created range start year</label>
           <%= date_range_start_year %>
-          <div class="invalid-feedback">Created year must be between <%= min_year %> and <%= max_year %></div>
+          <div class="invalid-feedback" data-date-validation-target="error"></div>
+          <div class="invalid-feedback" data-date-range-target="startError"></div>
```

Comment:
Each controller has its own place to report errors to avoid stepping on each other.